config/locales/en/layout.yml

```diff
@@ -28,6 +28,8 @@ en:
         title: 'Sidebar'
     course_users:
       title: 'Manage Users'
+    duplication:
+      title: 'Duplicate Data'
```

Comment:
For now yes. In future it'll allow cherry picking. I'll change it to `Duplicate Course` for now and change it again when the cherry picking goes in.